dotcms-integration/src/test/java/com/dotcms/rest/api/v1/template/TemplateResourceTest.java

```diff
@@ -1240,7 +1240,7 @@ public void legacyUUIDTemplate() throws DotDataException, DotSecurityException,
                 templateFromDaBaseBeforeUpdate.getDrawedBody());
         assertTrue(templateLayoutBeforeUpdate.existsContainer(container, uuid));
 
-        //Thread.sleep(10000);
+        //Thread.sleep(100xxzzzxxxxzzxzxzxzxzzxzzxzz00);
```

Comment:
delete this?

Reply:
done https://github.com/dotCMS/core/pull/28217/commits/2742109a15ec454afda9483bea248561df4ede5b#diff-745f7eccb02b77ba5b0bc21bba208f1fa5113e2e1b93261173f2091a579936b5L1243